src/ComputationContainer/Server/Helper/Helper.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <grpc/grpc.h>
+#include <grpcpp/security/server_credentials.h>
+#include <grpcpp/server.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/server_context.h>
+
+#include "LogHeader/Logger.hpp"
+
+/*
+ * @param &builder 呼び出し側で宣言したServerBuilder変数
+ * @param logSource ログ発行元を示す文字列（例："[Cc2CcForJob]", "[Cc2Cc]"）
+ * @param endpoint エンドポイント
+ */
+static void runServerCore(grpc::ServerBuilder &builder, std::string logSource, std::string endpoint)
```

Comment:
This doesn't make much difference, but I want.
```suggestion
static void runServerCore(grpc::ServerBuilder &builder, const std::string& logSource, const std::string& endpoint)
```

Reply:
@mdonaka Noted with thanks.  
I fixed it, please check that.

Reply:
I checked it. I hadn't thought of the format... Thanks for the fix!